### ISSUE TITLE
limit our snyk tests on travis to just merges on master

### ISSUE THREAD

**Issue by fczuardi** [CraveFood/farmblocks#182]
The free plan has a limit of 100 queries per month, and my guess is that we dont need all of them, we definitely dont need to test the dependencies on every push as we are doing now.

One way to lower our queries and still have a useful protection would be to run it once the pr is approved and merged into master.
